install_dependencies.bat

```diff
@@ -1,5 +1,8 @@
 @echo off
 
+if %1'==/?' goto :help
```

Comment:
let's keep this file simple: revert the changes in this file. then rebase to develop, please